libs/hook/useDeleteFollower.ts

```diff
@@ -0,0 +1,17 @@
+import { useMutation, useQueryClient } from "react-query";
+import deleteFollower from "../api/deleteFollower";
+
+const useDeleteFollower = () => {
+  const queryClient = useQueryClient();
+  const mutation = useMutation({
+    mutationFn: (login: string) => {
+      return deleteFollower(login);
+    },
+    onSuccess: () => {
+      queryClient.invalidateQueries(["get-follow-info"]);
+    },
+  });
+  return mutation;
```

Comment:
이 useMutation에서 사용되는것은 mutate 메서드이기 때문에,
return 자체를 mutation.mutate로 해줘도 좋을것 가타요!
```suggestion
  return mutation.mutate;
```